config/freertos/esp32/README.md

```diff
@@ -4,3 +4,54 @@
 - You can run idf targets like `menuconfig` or `monitor` by specifing the target as an argument to the `build_firmware.sh` script
 - The GPIO pins for the configured serial port can be set with `menuconfig` (see `micro-ROS Transport Settings` menu)
 - ESP32 only runs in singlecore mode (`CONFIG_FREERTOS_UNICORE=y` setting)
+
+
+# Notes for ESP32-CAMERA 
+
+- SPIRAM is requierd only for ESP32-CAMERA and can cause a fatal error in ESP32 (Enable in  `menuconfig Component config > ESP32-specific` OR  add `CONFIG_ESP32_SPIRAM_SUPPORT=y` to `sdkconfig.defaults` file).
+- Make sure that `rtc_gpio_desc` array is supported in `menuconfig: Component config > Driver configurations > RTCIO configuration` (set to TRUE)
+  OR write this line `CONFIG_RTCIO_SUPPORT_RTC_GPIO_DESC=y` to `sdk.defaults`.
+- Choose your camera pins configuration in `menuconfig Camera configuration > Camera pins`. (`BOARD_ESP32CAM_AITHINKER` is default).
+- In order to view the images you will need to create a cv_bridge:
```

Comment:
I would like to see this Python app as a `.py` file included in the `freertos_apps` app specific folder